software/fishstick-stm32f401/memory.x

```diff
@@ -0,0 +1,6 @@
+/* Linker script for the STM32F103C8T6 */
```

Comment:
Comment is wrong? Or does this need to be changed for the stm32f4xx?

Reply:
removed